### ISSUE TITLE
Investigate Web Search (Win+W)

### ISSUE THREAD

**Issue by rcmaehl** [rcmaehl/MSEdgeRedirect#175]
Related:
Windows 11 current stable release (2200.856)
Pressing windows key + W, brings up the web search, with a bunch of widgets showing it in by default (mainly news and weather)
if I simply type in a search than hit return, such as final fantasy.
It opens in chrome, with bing, despite default search engine and MSEedgeRedirect settings set to google. It does show it will do this (shows final fantasy - bing search)
below that link are some suggestions:
final fantasy 14
final fantasy 7
etc
If I instead click final fantasy 7, it does indeed use google. I get to this url as desired:
https://www.google.com/search?q=final%20fantasy%207&qs=AS&pq=final%20fantasy&sk=AS2&sc=5-13&cc=US&setLang=en-US&FORM=U627SB&pc=U627

Possibly unrelated to this app: Many, but not all of the default news widgets, if I click them , instead of going to the news article they usually go to a page like this:
https://www.google.com/url?q=https://www.yahoo.com/lifestyle/good-news-yellowstone-fans-season-204400722.html
The page you were on is trying to send you to https://www.yahoo.com/lifestyle/good-news-yellowstone-fans-season-204400722.html.

 If you do not want to visit that page, you can [return to the previous page](https://www.google.com/url?q=https://www.yahoo.com/lifestyle/good-news-yellowstone-fans-season-204400722.html#).

Useless page.. Probably I can fix these with a tampermonkey script, but yea weird it does that.

Neither a big deal as I don't care for this widge and figured it would be something startallback would disable.. But it does not so I left it on. I sometimes just block the key combo with autohotkey and have it do something else useful (minimize active window). But yea worth noting for ppl who do like this functionality.

_Originally posted by @Axer128 in https://github.com/rcmaehl/MSEdgeRedirect/issues/166#issuecomment-1232184802_

**Comment by Axer128** [comment]
Partially fixed in 0.7.1.0 👍  :
Installed via rls download, rclick run at as admin, active mode, same settings I was running before in 0.7.0.2.

If I Win+W: Click first news article (Fast food chains raise millions to oppose California wage law - WSJ. I get this error:
<img width="291" alt="image" src="https://user-images.githubusercontent.com/63986569/193441160-d9c1e51c-0029-4f7f-b7ff-6b5d9680f63e.png">
On previous version, these would open a fresh chrome instance (with chrome running already) with nothing in it. No errors.

Tried first 6 results, all same error. Nothing logged in %userprofile%\AppData\Local\MSEdgeRedirect\logs

Notable changes since testing a month ago: I upgraded to Win 11, 22H2 stable, current build: 22621.525. Somewhere along the lines I broke the ability to search in the win+W widgets box. It simply does not display a search box anymore, perhaps a privacy setting. Not sure how to turn it back on.
Anyways I can still search in Win+S:
In 0.7.0.2 , this was broken, same as above, opens a blank chrome instance.
Now it works 99.5% as expected:
Clicking on todays feature search image: Mahatma Ghandi's birthday works correctly, opens this url:
https://www.google.com/search?q=Gandhi%20Jayanti&setlang=en-us&cc=US
Click the 2nd one, "Life and work of the 'Great Soul', results in this url:
https://www.google.com/url?q=https://www.who.int/
Kinda useless, and doesn't auto redirect. 
Fixable with this userscript:
https://greasyfork.org/en/scripts/390770-workaround-for-google-i-m-feeling-lucky-redirect/code
Though im not confident it always gets the right article, as I think it just uses google ai im feeling lucky code, tho perhaps thats how microsoft programmed it, not sure. Anyways, low importance, issues probably M$ caused.


3rd link also behaves different: "Feats of activism"
it brings up this url in chrome:
https://www.google.com/search?q=historys-most-admirable-feats-of-activism+-site%3Amsn.com+-site%3Abing.com
Which does a search and does display the correct article because it's doing "-site:msn.com -site:bing.com" - which means it filters out any results from msn.com and bing.com . The correct article is on msn.com:
https://www.msn.com/en-gb/news/world/historys-most-admirable-feats-of-activism/ss-AA10oyEB
Removing both of the minues results in no results, as I guess google is trying to only you show you results that appear on BOTH sites, and its only on msn.com. Removing -site:bing and changing -site:msn.com to site:msn.com does get the correct url as top result.
4 links below (top 4 trending searches) all work as expected, search result in chrome on google.

win+s, than typing final fantasy, first result is a local file, so i press down, strike return, which shows the web results inside the ui, than return again, correctly shows the search in google on chrome. Same if I click the bing icon that says "open results in browser"
Inside the window under the highlighted "best match", the first result is jp.finalfantasy.com - clicking it goes to bing which does a few redirect eventually getting me to the correct url.
If I arrow down a couple to the first "result from the web" first result is the more expected finalfantasy.com,  2nd in the final fantasy wikipedia article. Both go to bing and redirect to correct pages.

TLDR:
Win+W still broken. Win+S works. In pretty good shape just some weird idiosyncrasies M$ caused. 

Related:
I made a script to auto redirect bing.com results to google.com since oddly couldn't find one on greasyfork, for the odd url M$ still manages to force to bing:
https://greasyfork.org/en/scripts/452361-bing-to-google-auto-redirect

**Comment by rcmaehl** [comment]
> First news article (Fast food chains raise millions to oppose California wage law - WSJ. I get this error:
> <img alt="image" width="291" src="https://user-images.githubusercontent.com/63986569/193441160-d9c1e51c-0029-4f7f-b7ff-6b5d9680f63e.png">

For that specific error try the latest test build. 
Since Microsoft doesn't include an actual link to the article until the page is loaded for an article, there's not much I can do outside of throwing the title into "I'm feeling lucky" logic. Google and DuckDuckGo both offer this but if there's others I'll gladly implement them.
